cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -434,6 +461,7 @@ func (s *Server) processQueue() {
 					sglog.Uint32("id", args.RepoID),
 					sglog.Strings("branches", branches),
 					sglog.Duration("duration", elapsed),
+					sglog.Duration("index delay", indexDelay),
```

Comment:
minor: I don't know if we document this anywhere, but probably best to use `_` for space and `.` for namespacing in attribute names. I see inconsistencies everywhere in our codebases, but atleast these can end up as OTEL attributes so those guidelines prefer that. Also makes it easier to process later when there aren't spaces.